examples/responseselectorbot/domain.yml

```diff
@@ -10,9 +10,6 @@ intents:
   - bot_challenge
   - chitchat
 
-actions:
```

Comment:
just to double-check: so the `utter_chitchat` action will get added automatically now as well? nice!

Reply:
Yes, it will be added [here](https://github.com/RasaHQ/rasa/pull/6591/files#diff-2182cd09fd656a4c8e34b0acfd8d91e1R285-R302)